workflow_manager.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/microsoft/BladeMonRT/nodes"
+	"github.com/microsoft/BladeMonRT/workflows"
+)
+
+/** Class for parsing workflow definitions. */
+type WorkflowManager struct {
```

Comment:
The more I look at WorkflowManager the more I don't like it's name. Let's rename it to `WorkflowFactory` because that's basically what it is

Reply:
Done